packages/material-ui/src/Tabs/ScrollbarSize.js

```diff
@@ -4,13 +4,11 @@ import EventListener from 'react-event-listener';
 import debounce from 'debounce'; // < 1kb payload overhead when lodash/debounce is > 3kb.
 
 const styles = {
-  width: 90,
-  height: 90,
+  width: 99,
+  height: 99,
   position: 'absolute',
-  top: -9000,
+  top: -9999,
   overflow: 'scroll',
-  // Support IE 11
-  msOverflowStyle: 'scrollbar',
```

Comment:
Why was this removed?

Reply:
Damn, I can't test the change on IE 11, the documentation is still broken. I have google the prefix, looked at Bootstrap, couldn't find the need. But yeah, better be safe, I'm reverting until we can test it properly.